recipes-kernel/linux/linux-nilrt_git.bb

```diff
@@ -7,4 +7,5 @@ require linux-nilrt.inc
 
 # This is the place to overwrite the source AUTOREV from linux-nilrt.inc, if
 # the kernel recipe requires a particular ref.
-#SRCREV = ""
+SRCREV = "378d8fbcc32082ee008a0df967ebd0f41a182877"
+SRCREV_xilinx-zynq = "051c9569fc919a173fbc7a56c75efdbba3b13b8c"
```

Comment:
This hash is for the nilrt/21.8/4.14 branch. The nilrt/22.8/4.14 branch head is at commit bc0d982eacfb9764302e888ea6d9f3d3bb4a5cc9